### ISSUE TITLE
[Bug] RomM cannot handle folders with 'é'

### ISSUE THREAD

**Issue by SatorSquare** [rommapp/romm#1293]
**RomM version**
v3.5.1

**Describe the bug**
If you have a folder that contains an `é` such as `SAM Coupé` in your library directory then RomM will choke with an error like: 

`UnicodeEncodeError: 'utf-8' codec can't encode character '\udce9' in position 1699: surrogates not allowed`

**To Reproduce**

1. Create a folder like `SAM Coupé` in your library
2. Restart RomM

**Expected behavior**
Ideally RomM would handle this without an issue, however if it cannot then a more clear error message would be helpful.

**Desktop (please complete the following information):**

- OS: Windows
- Browser: Chrome, Firefox

**Additional context**
One extra bit of relevancy here. The library was a CIFS volume mount connected to an SMB share on my network. If this cannot be reproduced locally, it's possible that is the extra wrinkle here.

**Comment by adamantike** [comment]
@SatorSquare, can you test if this keeps happening in version 3.6.0?

**Comment by SatorSquare** [comment]
@adamantike Yes, this still appears to be an issue in the latest version. However, the blast radius seems limited to this version's Platforms page. Previously, the entire app was unusable.

**Comment by cooperferguson42** [comment]
@gantoine This just occurred for me in v5.2.0. Can we confirm if it's not been fixed in 5.3.0-alpha.2?

**Comment by gantoine** [comment]
stop using é in folders and the problem goes away 😉
